<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>426</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>726</width>
    <height>402</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>726</width>
     <height>383</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="glMainWindow">
      <item row="0" column="1">
       <widget class="QTabWidget" name="twMainWindow">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabDBConnection">
         <attribute name="title">
          <string>DB connection</string>
         </attribute>
         <widget class="QListWidget" name="lwDrivers">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>191</width>
            <height>121</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
         </widget>
         <widget class="QPushButton" name="pbRefreshDriverList">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>160</y>
            <width>191</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Refresh driver list</string>
          </property>
         </widget>
         <widget class="QComboBox" name="cbODBCSourceData">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>60</y>
            <width>371</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="lODBCSourceData">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>30</y>
            <width>201</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ODBC Source Data</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="teODBCSourceData">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>90</y>
            <width>371</width>
            <height>91</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="pbSelectAFile">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>230</y>
            <width>191</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Select a file</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="teSelectAFile">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>260</y>
            <width>631</width>
            <height>51</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="lConnectorName">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>210</y>
            <width>121</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Connector name</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="leConnectorName">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>210</y>
            <width>251</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>[Persons$]</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pbConnect">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>320</y>
            <width>311</width>
            <height>30</height>
           </rect>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pbDisconnect">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>320</y>
            <width>301</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tabDBData">
         <attribute name="title">
          <string>DB data</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
